i8_terminal/commands/metrics/metrics_historical.py

```diff
@@ -245,7 +252,10 @@ def historical(
         .reset_index()
     )
     formatted_df["reversed_period"] = formatted_df.apply(
-        lambda row: f"{row.Period.split(' ')[1]} {row.Period.split(' ')[0]}", axis=1
+        lambda row: f"{row.Period.split(' ')[1]} {row.Period.split(' ')[0]}"
```

Comment:
Maybe its better to have this as a function (e.g. `reverse_period`) somewhere in utils.py for example as it is used in multiple commands. Also add a simple docstring in the function to make it clear that the function changes `Q1 2022` to `2022 Q1`.